charts/lakefs/templates/service.yaml

```diff
@@ -9,6 +9,9 @@ spec:
   ports:
     - port: {{ .Values.service.port }}
       targetPort: http
+{{- if .Values.service.nodePort }}
```

Comment:
please assert that `.Values.service.type == "NodePort"` to avoid issues for users configuring this.

Reply:
Added check for .Values.service.type